src/selectors/app.js

```diff
@@ -254,7 +255,11 @@ export const userHasPaid = state => {
   return getUserPaywallStatus(state) === PAYWALL_STATUS_PAID;
 };
 export const userCanExecuteActions = state => {
-  return userHasPaid(state) && !getKeyMismatch(state);
+  return (
+    userHasPaid(state) &&
+    !getKeyMismatch(state) &&
+    loggedInAsEmail(state) !== false
```

Comment:
Please, make sure to that the result ove `loggedInAsEmail(state)` is evaluated as a boolean if you want to detect falsy values. For example: `"" !== false` or `null !== false` is evaluated to `true`.
```suggestion
    !!loggedInAsEmail(state) !== false
```